macro/README.md

```diff
@@ -0,0 +1,100 @@
+# Helper macros for Host development 
+
+This crate provides a few helper macros for Host development.
+
+## Usage
+
+Add this to your `Cargo.toml`:
+
+```toml
+[dependencies]
```

Comment:
Should this maybe be `dev-dependencies`?

Reply:
Updated in https://github.com/nexus-xyz/nexus-zkvm/commit/b1b02fcafcfa15b5d3ee6c80c55a704117db836c